scripts/patterns-reorg/main.py

```diff
@@ -52,15 +54,19 @@ def write_guides_dir() -> None:
     (folder_path / "_toc.json").write_text(text)
 
 
+def generate_redirects() -> dict[str, str]:
```

Comment:
This naming is pretty similar to `determine_redirects`. Instead, you can do a module level constant `REDIRECTS = (*TOP_LEVEL_ENTRIES, *DELETED_PAGES)`